### PR TITLE
App header

### PR DESCRIPTION
**Pull request by poxvirus**

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -2,22 +2,39 @@
...
+      changeOrganization(organization) {
+        if(organization === null) return false;
+        if(this.activeOrg !== organization.login)
+          document.location.href = `/organizations/${organization.login}`;
+      },
+    },
+    beforeMount() {
+      let pathArray = window.location.pathname.split('/');
+      if(pathArray.length > 2 && pathArray[1] === 'organizations')
+        this.activeOrg = pathArray[2];
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: No magic number: 2                                                    no-magic-numbers

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -2,22 +2,39 @@
...
+    methods: {
+      changeOrganization(organization) {
+        if(organization === null) return false;
+        if(this.activeOrg !== organization.login)
+          document.location.href = `/organizations/${organization.login}`;
+      },
+    },
+    beforeMount() {
+      let pathArray = window.location.pathname.split('/');
+      if(pathArray.length > 2 && pathArray[1] === 'organizations')
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: Expected { after 'if' condition                                       curly<br>Expected space(s) after "if"                                          keyword-spacing<br>No magic number: 2                                                    no-magic-numbers

File: app/javascript/packs/application.js [hunk 3]
```diff
@@ -2,22 +2,39 @@
...
+    },
+    methods: {
+      changeOrganization(organization) {
+        if(organization === null) return false;
+        if(this.activeOrg !== organization.login)
+          document.location.href = `/organizations/${organization.login}`;
+      },
+    },
+    beforeMount() {
+      let pathArray = window.location.pathname.split('/');
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: 'pathArray' is never reassigned. Use 'const' instead                  prefer-const

File: app/javascript/packs/application.js [hunk 4]
```diff
@@ -2,22 +2,39 @@
...
       locale: 'es',
       messages: Locales.messages,
     }),
+    data: {
+      activeOrg: null,
+    },
+    methods: {
+      changeOrganization(organization) {
+        if(organization === null) return false;
+        if(this.activeOrg !== organization.login)
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: Expected { after 'if' condition                                       curly<br>Expected space(s) after "if"                                          keyword-spacing

File: app/javascript/packs/application.js [hunk 5]
```diff
@@ -2,22 +2,39 @@
...
     i18n: new VueI18n({
       locale: 'es',
       messages: Locales.messages,
     }),
+    data: {
+      activeOrg: null,
+    },
+    methods: {
+      changeOrganization(organization) {
+        if(organization === null) return false;
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: Expected space(s) after "if"                                          keyword-spacing

File: app/javascript/packs/application.js [hunk 6]
```diff
@@ -2,22 +2,39 @@
...
     el: '#app',
     i18n: new VueI18n({
       locale: 'es',
       messages: Locales.messages,
     }),
+    data: {
+      activeOrg: null,
+    },
+    methods: {
+      changeOrganization(organization) {
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: Expected to return a value at the end of method 'changeOrganization'  consistent-return

File: app/javascript/packs/application.js [hunk 7]
```diff
@@ -2,22 +2,39 @@
 
 import Vue from 'vue/dist/vue.esm';
 import VueI18n from 'vue-i18n';
-import Selector from '../selector.vue';
+import vSelect from 'vue-select';
 import Repository from '../repository.vue';
+import Dropdown from 'bp-vuejs-dropdown';
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: Absolute imports should come before relative imports                  import/imports-first

**Comment by poxvirus** [comment]
[Header 2.0](https://trello.com/c/euGBJ1SI/76-header-20)

File: app/assets/stylesheets/app/header.scss [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+    padding-left: -15px;
+  }
+
+  &__image {
+    width: 20px;
+    height: 20px;
+    margin-right: 10px;
+  }
+
+  .dropdown-toggle {
```
[the remark this answers is not in the document]
Comment (poxvirus) [reply app/assets/stylesheets/app/header.scss]: si

File: app/assets/stylesheets/app/header.scss [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+  &__selector {
+    width: 250px;
+  }
+
+  &__logo {
+    margin: 0;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
```
[the remark this answers is not in the document]
Comment (poxvirus) [reply app/assets/stylesheets/app/header.scss]: si

File: app/views/partials/_header.html.erb [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+        <div class="orgs-selector__option">
+          <img class="orgs-selector__image" :src="option.avatar_url"/>
+          {{ option.login }}
+        </div>
+      </template>
+    </v-select>
+
+  <% end %>
+  <%= link_to root_path, class: 'header__logo-container' do %>
+    <img class="header__logo" src="<%= image_path('froggo-logo.svg')%>"/>
```
[the remark this answers is not in the document]
Comment (poxvirus) [reply app/views/partials/_header.html.erb]: 👍

File: app/assets/stylesheets/app/header.scss [hunk 11]
```diff
@@ -0,0 +1,148 @@
...
+
+      &::before {
+        transform: rotate(315deg);
+      }
+    }
+  }
+
+  &__body {
+    background-color: $white;
+    border-left: solid 1px $gray;
```
[the remark this answers is not in the document]
Comment (poxvirus) [reply app/assets/stylesheets/app/header.scss]: 👍

File: app/assets/stylesheets/app/header.scss (hunk 9 above)
[the remark this answers is not in the document]
Comment (mf222) [reply app/assets/stylesheets/app/header.scss]: No te funcionó con `flex`? o dado que tiene tamaño fijo, centrarlo horizontal con `margin: 0 auto` y vertical con `align-self`? o no funcionó?

**Pull request merged**